lib/galaxy/tools/deps/resolvers/__init__.py

```diff
@@ -91,8 +92,12 @@ class NullDependency( Dependency ):
     dependency_type = None
     exact = True
 
+    def __init__(self, version, name):
+        self.version = version
+        self.name = name
+
     def shell_commands( self, requirement ):
         return None
 
 
-INDETERMINATE_DEPENDENCY = NullDependency()
+INDETERMINATE_DEPENDENCY = NullDependency
```

Comment:
Not sure if this change from a class instance to class alias is a good idea, there are still a number of places in `lib/` where the code is not updated to reflect this; also, if you need to change them all, then it would probably make sense to just use `NullDependency`.
@jmchilton: opinions?